profile/README.md

```diff
@@ -32,7 +32,7 @@ When you define your workflow, specify this action as `matlab-actions/run-comman
 ## Examples
 
 ### Run MATLAB Build on Self-Hosted Runner
-Starting in R2022b, the **Run MATLAB Build** action lets you run a build using the MATLAB build tool. You can use this action to run the tasks specified in a file named  `buildfile.m` in the root of your repository. For example, use a self-hosted runner to run the default tasks in your build plan as well as all the tasks on which they depend. To run the tasks, specify the **Run MATLAB Build** action in your workflow.
+Starting in R2022b, the **Run MATLAB Build** action lets you run a build using the MATLAB build tool. You can use this action to run the tasks specified in a file named  `buildfile.m` in the MATLAB current folder (returned by the [`pwd`](https://www.mathworks.com/help/matlab/ref/pwd.html) command) or its parent folders. For example, use a self-hosted runner to run the default tasks in your build plan as well as all the tasks on which they depend. To run the tasks, specify the **Run MATLAB Build** action in your workflow.
```

Comment:
This still seems a little bit off to me. Especially if we're talking about **Run MATLAB Build** it seems like a huge annoyance if users have to run a CI job just to check the default MATLAB current folder. I also still don't particularly like "MATLAB current folder" because at author time there is no running instance of MATLAB and thus no current folder which I think makes it both not quite accurate and somewhat confusing.

Reply:
Good point. 

I slightly changed this note text: "the MATLAB current folder is the same as the root of your repository. To specify a different current folder, use the `-sd` startup option or the `cd` command in your action." 

What do you think?